bulkExport/glueScripts/export-script.py

```diff
@@ -96,7 +104,7 @@
 
 # Drop fields that are not needed
 print('Dropping fields that are not needed')
-data_source_cleaned_dyn_frame = DropFields.apply(frame = filtered_dates_resource_dyn_frame, paths = ['documentStatus', 'lockEndTs', 'vid', '_references'])
+data_source_cleaned_dyn_frame = DropFields.apply(frame = filtered_dates_resource_dyn_frame, paths = ['documentStatus', 'lockEndTs', 'vid', '_references', '_tenantId', '_id'])
```

Comment:
Would it throw error or warning if _tenantId and _id does not exist?

Reply:
No, `DropFields` ignores fields that don't exist on the frame (I tested it)